docs/learn/learn-staking-miner.md

```diff
@@ -48,7 +51,28 @@ order to submit solutions to the NPoS election. When the correct time comes, it
 solution and submits it to the chain. The default miner algorithm is sequential Phragmén with a
 configurable number of balancing iterations that improve the score.
 
-![NPoS election optimization](../assets/staking-miner/NPoS-election-optimization.png)
+## Deposit and reward mechanics
+
+The staking miners are required to pay a deposit in order to post their solutions. The deposit is a
+sum of the `SignedDepositBase`, which is a fixed amount and `SignedDepositByte`, a variable amount
+per KB of solution data. All good solutions are subject to recieving a `SignedRewardBase`.
+
+Current deposit(**SignedDepositBase**) is
+{{ polkadot: <RPC network="polkadot" path="consts.electionProviderMultiPhase.signedDepositBase" defaultValue={16} filter="humanReadable"/> :polkadot }}
+{{ kusama: <RPC network="kusama" path="consts.electionProviderMultiPhase.signedDepositBase" defaultValue={16} filter="humanReadable"/> :kusama }}
+
+Current deposit per byte(**SignedDepositByte**) is
+{{ polkadot: <RPC network="polkadot" path="consts.electionProviderMultiPhase.signedDepositByte" defaultValue={16} filter="humanReadable"/> :polkadot }}
+{{ kusama: <RPC network="kusama" path="consts.electionProviderMultiPhase.signedDepositByte" defaultValue={16} filter="humanReadable"/> :kusama }}
+
+Current rewards(**SignedRewardBase**) is
+{{ polkadot: <RPC network="polkadot" path="consts.electionProviderMultiPhase.signedRewardBase" defaultValue={16} filter="humanReadable"/> :polkadot }}
+{{ kusama: <RPC network="kusama" path="consts.electionProviderMultiPhase.signedRewardBase" defaultValue={16} filter="humanReadable"/> :kusama }}
```

Comment:
```suggestion
Current deposit(**SignedDepositBase**) is
{{ polkadot: <RPC network="polkadot" path="consts.electionProviderMultiPhase.signedDepositBase" defaultValue={400000000000} filter="humanReadable"/> :polkadot }}
{{ kusama: <RPC network="kusama" path="consts.electionProviderMultiPhase.signedDepositBase" defaultValue={133333332000} filter="humanReadable"/> :kusama }}

Current deposit per byte(**SignedDepositByte**) is
{{ polkadot: <RPC network="polkadot" path="consts.electionProviderMultiPhase.signedDepositByte" defaultValue={97656} filter="humanReadable"/> :polkadot }}
{{ kusama: <RPC network="kusama" path="consts.electionProviderMultiPhase.signedDepositByte" defaultValue={32551} filter="humanReadable"/> :kusama }}

Current rewards(**SignedRewardBase**) is
{{ polkadot: <RPC network="polkadot" path="consts.electionProviderMultiPhase.signedRewardBase" defaultValue={10000000000} filter="humanReadable"/> :polkadot }}
{{ kusama: <RPC network="kusama" path="consts.electionProviderMultiPhase.signedRewardBase" defaultValue={100000000000} filter="humanReadable"/> :kusama }}
```

The RPC formatting you used here is perfect and won't be modified by prettier in any negative way!  Modifications are required for the default values to display properly before the on-chain values kick-in.  In the current state the page will initially loading with 0 for all these values until the RPC values are retrieved.  The proper defaults should be the raw values (without commas) you would get when calling those endpoint paths say via PolkadotJS, prior to applying the filter. The 'humanReadable` filter will take that raw value of say `400000000000` Plancks and convert it to `40 DOT` prior to rendering.  So for example:

![image](https://user-images.githubusercontent.com/13341935/196601589-5556ba3c-cdcf-4889-a9a7-808a0b2c447e.png)

Let me know if you have any further questions on this!

Reply:
With the current precision of the rendered DOT values, the deposit byte value will be shown as 0.000 DOT. Not sure if we have to increase the precision which effects the whole WIki or make an exception just for this one case!

Reply:
Would be good if you also show an example: a solution that is 200KB of data to post onchain, costs xxx. 

You can look at subscan and old block to find the size of a typical transaction.

Reply:
I'm going to create another issue for adding this feature, meanwhile, for the `signedDepositByte` value I will put the right output into the default answer and leave a TODO above the code block.